tutor/specs/e2e/assignment-review.e2e.js

```diff
@@ -0,0 +1,174 @@
+import { visitPage, setTimeouts } from './helpers'
+
+// the BE mock api server is primarily in backend/task-plans
+
+describe('Assignment Review', () => {
+
+  beforeEach(async () => {
+    await setTimeouts()
+
+    await visitPage(page, '/course/1/assignment/review/2')
+    await page.route('/api/plans/*', req => req.continue())
+  });
+
+  it('loads and views feedback', async () => {
+    await page.click('testEl=submission-overview-tab')
+    await expect(page).toHaveSelector('testEl=student-free-responses')
+  });
+
+  it('loads and views scores', async () => {
+    await page.click('testEl=assignment-scores-tab')
+    await expect(page).toHaveSelector('testEl=scores')
+  });
+
+  it('can grade WRM', async () => {
+    await page.route('/api/courses/1/grading_templates*', req => req.continue())
+    await page.click('testEl=grade-answers-btn')
+    // if this times out, restart the backend server
+    // the specs may have ran to many times and graded all the responses
```

Comment:
@nathanstitt how is this going to affect the gh-actions? would it timeout and fail the test?

Reply:
no, since that only runs once.  It becomes and issue when you're running the tests with `--watch`, that causes them to re-run whenever you make a change.  Eventually the fake BE will run out of non-excluded exercises.